docs/voldoen-aan-wetten-en-regels/hulpmiddelen/standaarden.md

```diff
@@ -0,0 +1,144 @@
+
+Er zijn verschillende soorten standaarden, zoals normen, Technische Rapporten (TR) en Technische Specificaties (TS).
+
+- Normen zijn formele, breed geaccepteerde afspraken, vaak ontwikkeld door internationale organisaties zoals ISO/IEC.
+- Technische Rapporten (TR) hebben een meer informatief karakter. Een TR wordt gebruikt om informatie te delen die niet geschikt is voor een norm, bijvoorbeeld gegevens uit onderzoeken of enquêtes. Een TR is vaak de opmaat voor het opstellen van een norm.
+- Technische Specificaties (TS) zijn normatief, en bieden voorlopige specificaties, vooral voor technologieën in ontwikkeling. Een TS kan in een later stadium worden omgezet in een volledige norm als er voldoende consensus voor is.
+
+!!! tip "Nog geen norm, maar wel al bruikbaar"
+
+    Zowel een Technisch Rapport als een Technische Specificatie bieden dus ondersteuning in situaties waar nog geen norm bestaat of waar consensus nog niet is bereikt. Dus een TS of TR kan ook zeker interessante informatie bevatten voor organisaties, maar ze hebben niet de status of juridische binding van een norm.
+
+Daarnaast is er onderscheid tussen internationale normen (ISO), Europese normen (EN) en Nederlandse normen (NEN). De NEN onderscheidt ook [verschillende typen afspraken](https://www.nen.nl/verschillende-typen-afspraken), zoals Nederlandse praktijkrichtlijn (NPR) of Nederlandse voornorm (NVN).
+
+Europese normen zijn essentieel voor de AI Verordening, omdat ze geharmoniseerde standaarden bieden die juridische zekerheid verschaffen, met name voor hoog-risico AI-systemen. Veel Europese normen beginnen als internationale ISO-normen en worden vervolgens aangepast aan de Europese context.
+
+## Waarom zijn standaarden belangrijk voor de AI-verordening?
+Wanneer AI-systemen worden ontworpen of gebruikt, bieden standaarden praktische richtlijnen en technische kaders (praktische meerwaarde) waarmee compliance aantoonbaar kan worden gemaakt (juridische meerwaarde).
+
+### Juridische waarde van standaarden
+Als een organisatie zich houdt aan de bij de wet behorende _geharmoniseerde Europese normen_, wordt aangenomen dat er wordt voldaan aan de AI-verordening (_Presumption of Conformity_). Dit betekent dat het AI-systeem wordt beschouwd als conform de wettelijke vereisten, wat zorgt voor juridische zekerheid en eenvoudiger toezicht​.
+Hoewel het gebruik van standaarden vrijwillig is, bewijst naleving ervan dat producten en diensten een bepaald niveau van kwaliteit, veiligheid en betrouwbaarheid bereiken. In sommige gevallen worden standaarden in de wetgeving genoemd als een voorkeursmethode of zelfs als een verplichte vereiste om aan specifieke wetten te voldoen, zoals veiligheidswetgeving of interoperabiliteitsvereisten. 
```

Comment:
Dit is nu algemeen geformuleerd, maar ik zou het specifiek maken voor de AI-verordening.
De geharmoniseerde normen zijn niet wettelijk verplicht, maar dankzij de presumption of conformity wel juridisch bindend. Organisaties mogen er voor kiezen om de geharmoniseerde norm níet te implementeren, maar moeten dan onderbouwen dat hun eigen standaard op adequate wijze invulling geeft aan de essentiële eisen. Dit kost tijd en geld, en er is schaarse expertise voor nodig. Het geeft bovendien juridische onzekerheid. De verwachting is dat verreweg de meeste organisaties daarom de geharmoniseerde normen zullen volgen.